CHANGELOG.md

```diff
@@ -60,8 +60,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-- The MSRV is now 1.56.1
-  ([#1792](https://github.com/nix-rust/nix/pull/1792))
+- The epoll interface now uses a type.
```

Comment:
You need to rebase on master, and then move the CHANGELOG stuff to the top of the file.

Reply:
Rebased.